src/Raven.Client/Constants.cs

```diff
@@ -220,6 +220,8 @@ private Querying()
                 }
 
                 public const string IntersectSeparator = " INTERSECT ";
+
+                public const string DynamicQueryPrefix = "dynamic/";
```

Comment:
On client-side user can supply us with indexName,
e.g. `session.Query<dynamic>("<index-name-here>")`
we need to figure out if this is dynamic query or query against static index, so I left the `dynamic/` prefix as it was before. This is not send to the server, just to check if query is dynamic or not.